From 6d06678a76d9275beb48917d7653ea6a610c7717 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Micha=C5=82=20G=C3=B3rny?= <mgorny@gentoo.org>
Date: Sun, 23 Jan 2022 15:03:35 +0100
Subject: [PATCH] Update test_invalid_coverage_source for coverage-6.2

Update test_invalid_coverage_source to make the "No data was collected"
less strict, as the output has changed in coverage-6.2.  This solution
was suggested by Tom Callaway (@spotrh) on the linked bug.

Fixes #509
---
 tests/test_pytest_cov.py | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/tests/test_pytest_cov.py b/tests/test_pytest_cov.py
index f93d352..6d97949 100644
--- a/tests/test_pytest_cov.py
+++ b/tests/test_pytest_cov.py
@@ -981,7 +981,7 @@ def test_invalid_coverage_source(testdir):
         '*10 passed*'
     ])
     result.stderr.fnmatch_lines([
-        'Coverage.py warning: No data was collected.*'
+        '*No data was collected.*'
     ])
     result.stdout.fnmatch_lines([
         '*Failed to generate report: No data to report.',
-- 
2.34.1

